src/main/java/bio/terra/app/configuration/ElasticSearchConfiguration.java

```diff
@@ -1,21 +0,0 @@
-package bio.terra.app.configuration;
-
-import org.apache.http.HttpHost;
-import org.elasticsearch.client.RestClient;
-import org.elasticsearch.client.RestClientBuilder;
-import org.elasticsearch.client.RestHighLevelClient;
-import org.springframework.beans.factory.annotation.Value;
-import org.springframework.context.annotation.Bean;
-import org.springframework.context.annotation.Configuration;
-
-@Configuration
-public class ElasticSearchConfiguration {
-
-  @Bean
-  public RestHighLevelClient client(
-      @Value("${elasticsearch.hostname}") String hostname,
-      @Value("${elasticsearch.port}") int port) {
```

Comment:
With these removed you should also remove the corresponding lines in `application.properties`:
```
features.search.api=disabled
elasticsearch.hostname=localhost
elasticsearch.port=9200
elasticsearch.numShards=3
```

Reply:
Good shout, plus I wonder if these properties were overridden per environment too?  That could be another opportunity for clean-up.

Reply:
Thanks for catching that!